server/testdata/webhook-server-issue-updated-commented-mention-user.json

```diff
@@ -0,0 +1,606 @@
+                            },
+                            "displayName": "Lev Brouk",
+                            "active": true,
+                            "timeZone": "Etc/UTC"
+                        },
+                        "body": "unik with mentioned user [~user]",
+                        "updateAuthor": {
+                            "self": "http://sales-jira.centralus.cloudapp.azure.com:8080/rest/api/2/user?username=levbrouk",
+                            "name": "levbrouk",
+                            "key": "levbrouk",
+                            "emailAddress": "lev@mattermost.com",
+                            "avatarUrls": {
+                                "48x48": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=48",
+                                "24x24": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=24",
+                                "16x16": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=16",
+                                "32x32": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=32"
+                            },
+                            "displayName": "Lev Brouk",
+                            "active": true,
+                            "timeZone": "Etc/UTC"
+                        },
+                        "created": "2019-06-04T00:15:17.234+0000",
+                        "updated": "2019-06-04T00:15:17.234+0000"
+                    }
+                ],
+                "maxResults": 10,
+                "total": 10,
+                "startAt": 0
+            },
+            "votes": {
+                "self": "http://sales-jira.centralus.cloudapp.azure.com:8080/rest/api/2/issue/PRJX-14/votes",
+                "votes": 0,
+                "hasVoted": false
+            },
+            "worklog": {
+                "startAt": 0,
+                "maxResults": 20,
+                "total": 0,
+                "worklogs": []
+            }
+        }
+    },
+    "comment": {
+        "self": "http://sales-jira.centralus.cloudapp.azure.com:8080/rest/api/2/issue/10013/comment/10082",
+        "id": "10082",
+        "author": {
+            "self": "http://sales-jira.centralus.cloudapp.azure.com:8080/rest/api/2/user?username=levbrouk",
+            "name": "levbrouk",
+            "key": "levbrouk",
+            "emailAddress": "lev@mattermost.com",
+            "avatarUrls": {
+                "48x48": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=48",
+                "24x24": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=24",
+                "16x16": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=16",
+                "32x32": "https://www.gravatar.com/avatar/8c504b6c8f13cf9dc9ffb8af57323320?d=mm&s=32"
+            },
+            "displayName": "Lev Brouk",
+            "active": true,
```

Comment:
Please ensure that all this test data does not contain any real sensitive or confidential data.

Reply:
copied from same already present file.